ee/server/service/teams.go

```diff
@@ -144,7 +144,10 @@ func (svc *Service) ModifyTeamAgentOptions(ctx context.Context, teamID uint, opt
 
 	if options != nil {
 		team.Config.AgentOptions = &options
+		// TODO(mna): validate agent options before saving
```

Comment:
It's fine to keep those TODOs in the PR, they will be addressed shortly in a subsequent PR (validation of the settings) and they help me identify where to add this validation hook.